lib/danger/danger_core/plugins/dangerfile_git_plugin.rb

```diff
@@ -46,7 +46,7 @@ def self.instance_name
 
     def initialize(dangerfile)
       super(dangerfile)
-      raise unless dangerfile.env.scm.instance_of?(Danger::GitRepo)
+      raise unless dangerfile.env.scm.class == Danger::GitRepo # rubocop:disable Style/ClassEqualityComparison
```

Comment:
Addressing this offense would change the behavior.
It actually causes the test to fail, so it is disabled: https://github.com/danger/danger/actions/runs/3508633313/jobs/5877172874